scripts/fixes.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
+
```

Comment:
Safer with /bin/bash, it should be available on all modern linux systems
```suggestion
#!/bin/bash

```

Reply:
The env stuff is to make it work on Macs. It sucks.

Reply:
I see, then we keep it